core/src/main/resources/jenkins/security/QueueItemAuthenticatorMonitor/message.groovy

```diff
@@ -0,0 +1,65 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2019, CloudBees, Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+package jenkins.security.QueueItemAuthenticatorMonitor
+
+def f = namespace(lib.FormTagLib)
+
+div(class: "alert alert-warning", role: "alert") {
+
+    form(method: "post", action: "${rootURL}/${my.url}/act") {
+        f.submit(name: 'redirect', value: _("Learn more…"))
```

Comment:
`Learn more…` gives me:

![screenshot_2019-03-01_115551_001](https://user-images.githubusercontent.com/2662497/53634147-b335b080-3c19-11e9-8f1d-ab55c08d14b6.png)

And `...` is working fine.

Reply:
Hmm, all source files except `*.properties` are expected to be in UTF-8, so if so that sounds like a bug in `stapler-groovy`.